mica-web-model/src/main/java/org/obiba/mica/web/model/NetworkDtos.java

```diff
@@ -227,6 +227,9 @@ Network fromDto(@NotNull Mica.NetworkDtoOrBuilder dto) {
     else
       network.setModel(new HashMap<>());
 
+    if (dto.hasMaelstromAuthorization())
+      network.getModel().put("maelstromAuthorization", AuthorizationDtos.fromDto(dto.getMaelstromAuthorization()));
```

Comment:
This field did not exist in network's form for mica2 1.x (see online mica demo), so there is no reason for handling this.